docs/programmatic-usage.md

```diff
@@ -0,0 +1,60 @@
+# Programmatic Usage
+
+Everything you can do through the CLI, you can do programmatically (and more!).
+
+## Starting a dev server
+
+The simple method:
+
```

Comment:
```suggestion
```

Contrasting with the "complex" use case pushes the average person to think about the complex use case. But as I understand it the average person shouldn't think about the complex use case. So I think it is best to remove this line.